ldap/servers/plugins/acctpolicy/acct_util.c

```diff
@@ -27,7 +27,7 @@ Hewlett-Packard Development Company, L.P.
 #include "slapi-plugin.h"
 
 /* Globals */
-static void *plugin_id = NULL;
+static void *global_plugin_id = NULL;
```

Comment:
Worth checking with globals we have ODR turned on.

Reply:
No ODR errors